app/models/gemini_connection.rb

```diff
@@ -90,6 +95,12 @@ def sync_connection!
       recipient = Gemini::RecipientId.find_or_create(token: access_token)
       update(recipient_id: recipient.recipient_id)
       update_default_currency
+
+      # Add bitFlyer deposit id to each of the publisher's channels
+      publisher.channels.each do |channel|
+        channel_recipient = Gemini::RecipientId.find_or_create(token: access_token, label: channel.id)
+        channel.update_column(:gemini_recipient_id, channel_recipient.recipient_id)
+      end
```

Comment:
Some creators have hundreds of channels. May we think of a better way to do this?

Reply:
I can make it a job, sure.